### DISCUSSION TITLE
License of entries we add to the repo

### DISCUSSION THREAD

**Discussion by marabuuu** [NFDI4BIOIMAGE/training#553] [General]
Hi all, 

Did we define somewhere which licenses we allow for the ressources we add into our yml file?  Are we for example also allowing CC BY-NC-SA 4.0?

I did just find this sentence in the readme:
`All contents of this Jupyter book and the corresponding Github repository are licensed [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/), unless mentioned otherwise.`

Best,
Mara

**Comment by haesleinhuepf** [comment]
Hi @marabuuu ,

as we are just adding links and metadata (and not the actual content), we can add everything. The formulation "unless mentioned otherwise" ensures we are on the safe side.

Best,
Robert

**Reply by marabuuu** [reply]
Ok great, thanks for your quick reply @haesleinhuepf
